week2/week2.py

```diff
@@ -0,0 +1,116 @@
+# 1.Write a function called fizz_buzz that takes a number
+def fizzbuzz(num):
+    if num % 3 == 0 and num % 5 == 0:
+        return "fizz buzz"
+    elif num % 3 == 0:
+        return "fizz"
+    elif num % 5 == 0:
+        return "buzz"
+    else:
+        return num
+
+
+print(fizzbuzz(60))
+
+# 2. try and except block to avoid IndexError.
+num_list = [5, 10, 20]
+try:
+    if num_list[5] == 20:
+        print("number is the list")
+except IndexError:
+    print("List index out of range")
+
+
+# 3. Create a class with arguments
+class JetInventory():
+    def __init__(self, name, country):
+        self.name = name
+        self.country = country
+
+    def jet_details(self):
+        print(f'{self.name} {self.country}')
+
+
+jetInventory1 = JetInventory("AirIndia", "India")
+jetInventory1.jet_details()
```

Comment:
Above for `num_list` you are using `snake_case`, but now for `jetInventory1` you are using `camelCase`, be consistence, use either one, but then always